demos/hr_agent/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.10 AS base
+
+FROM base AS builder
+
+WORKDIR /src
+
+COPY requirements.txt /src/
+COPY workforce_data.json /src/
+RUN pip install --prefix=/runtime --force-reinstall -r requirements.txt
+
+COPY . /src
```

Comment:
ideally we should only copy files that are needed to be in image like copy only main.py and workforce_data.json

Reply:
Sure..